content/pl/docs/contribute/localization-pl.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Tłumaczenie dokumentacji na język polski
+content_template: templates/concept
+card:
+  name: contribute
+  weight: 35
+  title: Tłumaczenie dokumentacji na język polski
+---
+
+{{% capture overview %}}
+
+Na tej stronie znajdziesz wskazówki i wytyczne przydatne przy tłumaczeniu dokumentacji Kubernetes na język polski.
+
+Dokumentem nadrzędnym jest angielski [opis stylu dokumentacji](/docs/contribute/style/style-guide).
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Wskazówki ogólne
+
+Staramy się, aby styl tłumaczenia był jak najbardziej naturalny. W przypadku dokumentacji technicznej może być to trudne zadanie,
+kiedy chcemy utrzymać precyzję tłumaczenia, ale zależy nam na unikaniu sytuacji, kiedy tekst zaczyna sprawiać wrażenie przetłumaczonego maszynowo.
+
+Pamiętajmy też, że oficjalna wykładnia zawsze znajduje się w tekście angielskim. Polskie tłumaczenie ma ułatwić pierwsze kroki osobom,
+które zaczynają swoją przygodę z Kubernetesem.
+
+## Wytyczne szczegółowe
+
+### Odmiana terminu Kubernetes
+
+Kubernetes jest nazwą własną, liczba pojedyncza, rodzaj męski. Odmieniamy: Kubernetesa, Kubernetesem itp.
+W uzasadnionych przypadkach można stosować też "system Kubernetes".
+
+### Odmiana terminów Pod, Deployment
+
+Odmieniamy zgodnie z ogólnymi zasadami - poda, deploymentu itp.
+
+### Ujednolicony słownik
+
+W sieci dostępne są słowniki terminów informatycznych. Poniższa tabela zawiera słowa specyficzne dla Kubernetesa i inne często używane wyrażenia.
+
+Termin angielski | Tłumaczenie
+-----|-----
+Container | Kontener
+control plane | warstwa sterowania
+Deployment | Deployment
+horizontal scaling | skalowanie horyzontalne
+Pod | Pod
```

Comment:
```suggestion
pod | pod
```

Reply:
Pod is often written with initial caps in English; it's a Kubernetes resource.